qirlib/Cargo.toml

```diff
@@ -28,6 +28,10 @@ lazy_static = "1.4.0"
 mut_static = "5.0.0"
 bitvec = "1.0.0"
 
+# TODO: Avoid a transitive unicode-ident dependency for now due to licensing.
+syn = "1.0, < 1.0.95"
```

Comment:
@samarsha I don't understand. Does the TODO mean we still need to work around the issue, or that this is a workaround for that issue?

Reply:
The version upper bounds for syn and proc-macro2 is the workaround, the todo is to remove the bounds when we can allowlist the license.

Reply:
Updated the comment.